package.json

```diff
@@ -152,13 +152,15 @@
     "wherearewe": "^2.0.1"
   },
   "devDependencies": {
+    "@libp2p/interfaces": "^3.0.3",
```

Comment:
without this, `ipfs-core-types` was failing on a lack of types from `@libp2p/interfaces/events` from the line `import type { EventHandler } from '@libp2p/interfaces/events'`